Services/DataBaseService/Commands/CreateUserCommands.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DataBaseService.Interfaces;
+using DTO;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace DataBaseService.Commands
+{
+    public class CreateUserCommand : ICommand<UserEmailPassword>
+    {
+        private readonly IRepository<UserEmailPassword> userRepository;
+
+        public CreateUserCommand([FromServices] IRepository<UserEmailPassword> userRepository)
+        {
+            this.userRepository = userRepository;
+        }
+
+        public void Execute(UserEmailPassword data)
+        {
+            //TODO Change to custom exception
+            if (string.IsNullOrEmpty(data.Email) || string.IsNullOrEmpty(data.PasswordHash))
+            {
+                throw new Exception("Not correct data");
+            }
```

Comment:
Change to Custom Exception

Reply:
this fale was delete from develop and I return it

Reply:
So? Why don't we make it better ?

Reply:
I won't change this code because it delete in develop now